Automattic/wp-calypso
# Jetpack connect activate dialog

In jetpack connect we need to redirect the users to their wp-admin when they already have jetpack installed in their sites, but they need to activate it before continue. We are going to show a dialog explaining how to do it before redirecting them back there.

this is how it looks like in this PR:
![image](https://cloud.githubusercontent.com/assets/1554855/14604134/3a720742-0572-11e6-8f1a-94a99da940d5.png)


Keep in mind that the image has been done by me, so probably @rickybanister will want to kill it with fire and provide a proper one, this is only a placeholder :)

How to test
========
1. You need a site with jetpack disabled
2. Go to http://calypso.localhost:3000/jetpack/connect and enter the url of that site
3. You should see the dialog
4. Once you click to "activate", you should be redirected to [yoursite/wp-admin/plugins

cc/ @roccotripaldi @rickybanister
